tools/run_tests/xds_k8s_test_driver/framework/xds_flags.py

```diff
@@ -127,6 +127,13 @@
     default=False,
     help="Force resource cleanup, even if not created by this test run")
 
+COLLECT_APP_LOGS = flags.DEFINE_bool(
+    'collect_app_logs',
+    default=False,
```

Comment:
Why not turn this on by default?

Reply:
I don't think we need this in most cases on local dev. It'll write to "/tmp` by default (this is an absl thing). It won't make sense for pre-provisioned pods (like url map tests) either.
Also, `--log_dir` is different per each environment (kokoro/staging/local), so I want setting `collect_app_logs` to be a conscious choice.
And there's always flagfiles to set this flags if needed.